tests/containers/push_container_image_to_acr.pm

```diff
@@ -9,16 +9,21 @@
 
 use Mojo::Base 'publiccloud::basetest';
 use testapi;
-use containers::urls 'get_suse_container_urls';
+use containers::urls qw(get_suse_container_urls get_urls_from_var);
 
 sub run {
     my ($self, $args) = @_;
 
     $self->select_serial_terminal;
 
     my $provider = $self->provider_factory(service => 'ACR');
-    my ($untested_images, $released_images) = get_suse_container_urls();
-    my $image = $untested_images->[0];
+
+    my $image;
+    unless ($image = get_urls_from_var('CONTAINER_IMAGES_TO_TEST')->[0]) {
```

Comment:
Seeing the exact same routine in 3 different files kinda motivates to extract a subroutine for it, WDYT?

Reply:
Yes, the subroutine is called `get_urls_from_var` and already exists.